core/server/master/src/main/java/alluxio/web/WebInterfaceGeneralServlet.java

```diff
@@ -226,9 +228,10 @@ private void populateValues(HttpServletRequest request) {
 
   private void setUfsAttributes(HttpServletRequest request) {
     FileSystemMaster fsMaster = mMasterProcess.getMaster(FileSystemMaster.class);
-    Map<String, MountPointInfo> mountPoints = fsMaster.getMountTable();
-    MountPointInfo mountInfo = mountPoints.get(MountTable.ROOT);
-    if (mountInfo == null) {
+    MountPointInfo mountInfo;
+    try {
+      mountInfo = fsMaster.getMountPointInfo(new AlluxioURI(MountTable.ROOT));
+    } catch (InvalidPathException e) {
```

Comment:
log the exception, and consider catching `RuntimeException` or `Throwable` - any exception that we allow to propagate from here will crash the UI

Reply:
What about catch all the exceptions? Thanks for the info!
```
    MountPointInfo mountInfo;
    try {
      mountInfo = fsMaster.getMountPointInfo(new AlluxioURI(MountTable.ROOT));
    } catch (Exception e) {
      LOG.error("Unable to get root mount info", e);
      return;
    }
```

Reply:
Double checking, is it intentional to let `Error` propagate here?

Reply:
I checked all the web interface and this is the only place that use log.error. Other places all use throw Exception. Therefore, I change the log to throw Exception. Missing root mount information is not a small exception that we can ignore. Thanks for catching this!!

Reply:
Change to throw throwable in this place and add a jira https://alluxio.atlassian.net/browse/ALLUXIO-3354 about changing from throwing exceptions to logging.